src/popup/pages/NetworkForm.vue

```diff
@@ -186,35 +185,34 @@ export default defineComponent({
     }
 
     async function addOrUpdateNetwork() {
-      if (await $validator.validateAll()) {
-        store.commit('setUserNetwork', {
-          ...newNetwork.value,
-          index: newNetwork.value.index,
-        });
-        await selectNetwork(newNetwork.value.name);
-        goBack();
-      }
+      store.commit('setUserNetwork', {
+        ...newNetwork.value,
+        index: newNetwork.value.index,
```

Comment:
is the `index` property needed here?

Reply:
Yes I believe `index` is needed when we want to edit an existing network